### PR TITLE
Added a notice that only registered users can get a certificate

### PR DESCRIPTION
**Pull request by dodlin**

### CODE REVIEW

File: client/templates/find_certificate.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
         <form action="{% url 'client_find_certificate' %}" method="post">
             {% csrf_token %}
             {{ form|bootstrap }}
+            רק משתמשים רשומים במערכת זכאים להנפיק תעודה <br>
```
Comment (stavlocker) [inline client/templates/find_certificate.html:15]: * Move the text to the top of the form, so it's clearer
* The volunteers aren't "משתמשים", and a link to the volunteer form would be nice too. I'd change the text to "עוד לא נרשמתם כמתנדבים? לחצו כאן" - this makes it clear this form is only for registered volunteers
 * Wrap the text in <p> tag so it's not floating, maybe enlargen the text a little.

**Comment by stavlocker** [comment]
@dodlin Sorry for the code snatch, but there was something very small I wanted to change and then noticed another small thing.
I fixed them but I'll write it down here for future notice:

 * Don't use hardcoded URLs in templates, use the {% url %} tag 
 * <div>s are more for content blocks, I put the text in a <p> instead (<span> is also acceptable depending on the use case)

**Comment by dodlin** [comment]
> @dodlin Sorry for the code snatch, but there was something very small I wanted to change and then noticed another small thing.
> I fixed them but I'll write it down here for future notice:
> 
> * Don't use hardcoded URLs in templates, use the {% url %} tag
> * s are more for content blocks, I put the text in a  instead ( is also acceptable depending on the use case)
> 
> Because I made these two changes myself instead of giving comments about them, if you disagree with the changes I made say so and I'll restore the branch.
> 
> If it looks okay, you can close the original issue.

No problem :)
I tried to add the url with the template but it didn't work so I hard-coded it..

**Comment by stavlocker** [comment]
@dodlin Okay, cool, you can check out what I did if you want. I'm closing the original issue :)

**Pull request merged**